bool command
---
string result
string message
